browser/js/addProduct/addProduct.js

```diff
@@ -11,8 +11,8 @@ app.config(function ($stateProvider) {
 	});
 })
 
-app.controller('addProductCtrl',function($scope,AuthService,products,ProductFactory){
-	$scope.addProduct = ProductFactory.addProduct;
+app.controller('addProductCtrl',function($scope,AuthService,products,CartFactory){
```

Comment:
AuthService is to check if the user is an admin or not and whether they have permission to add a product